src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -368,8 +368,10 @@ private function getOrderProducts(Order $order): OrderProductsForViewing
             $product['quantity_refundable'] = $product['product_quantity'] - $product['product_quantity_return'] - $product['product_quantity_refunded'];
             $product['amount_refundable'] = $product['total_price_tax_excl'] - $product['total_refunded_tax_excl'];
             $product['amount_refundable_tax_incl'] = $product['total_price_tax_incl'] - $product['total_refunded_tax_incl'];
-            $product['displayed_max_refundable'] = $taxCalculationMethod === PS_TAX_EXC ? $product['amount_refundable'] : $product['amount_refundable_tax_incl'];
-            $resumeAmountKey = $taxCalculationMethod === PS_TAX_EXC ? 'total_refunded_tax_excl' : 'total_refunded_tax_incl';
+            /* $product['displayed_max_refundable'] = $taxCalculationMethod === PS_TAX_EXC ? $product['amount_refundable'] : $product['amount_refundable_tax_incl'];
+            $resumeAmountKey = $taxCalculationMethod === PS_TAX_EXC ? 'total_refunded_tax_excl' : 'total_refunded_tax_incl'; */
+            $product['displayed_max_refundable'] = $order->getTaxCalculationMethod() ? $product['amount_refundable'] : $product['amount_refundable_tax_incl'];
```

Comment:
This produces an error (in admin context this field is not correctly set in Order, so the amount was always displayed with taxes) that was fixed using `$taxCalculationMethod` (which is correctly set)

Reply:
done